### ISSUE TITLE
Unable to enable custom auth plugin on Windows

### ISSUE THREAD

**Issue by Obito93** [hashicorp/vault#6842]
**Describe the bug**
Trying to enable a self-written auth method results in vault on Windows results in the following error code:
- via CLI:
C:\vault>vault auth enable -path=example -plugin-name=example-auth-plugin plugin
Error enabling example-auth-plugin auth: Error making API request.

URL: POST https://127.0.0.1:8200/v1/sys/auth/example
Code: 400. Errors:

* exec: "C:\\vault\\vault_plugins\\sqlLoginPlugin": file does not exist
(EDIT: these should be double "\\". It seems github escapes them... see the provided screenshots)
Response via HTTP API:
{
    "errors": [
        "exec: \"C:\\\\vault\\\\vault_plugins\\\\sqlLoginPlugin\": file does not exist"
    ]
}
![bug](https://user-images.githubusercontent.com/32833974/59035487-af937480-886d-11e9-9117-48533fa4dfa0.PNG)
![bug2](https://user-images.githubusercontent.com/32833974/59035563-d6ea4180-886d-11e9-85bf-69c42befb0bd.PNG)

It seems there is an escaping issue

**To Reproduce**
-

**Expected behavior**
-

**Environment:**
* Vault Server Version (retrieve with `vault status`):
C:\vault>vault status
Key             Value
---             -----
Seal Type       shamir
Initialized     true
Sealed          false
Total Shares    1
Threshold       1
Version         1.1.2
Cluster Name    vault-cluster-ceed1c1b
Cluster ID      e5435a20-c94d-14ce-6f66-b90a5ba92ab3
HA Enabled      false

* Vault CLI Version (retrieve with `vault version`):
C:\vault>vault version
Vault v1.1.2 ('0082501623c0b704b87b1fbc84c2d725994bac54')

* Server Operating System/Architecture:
Windows 10 Build 17763

Vault server configuration file(s):

```hcl
storage "file" {
  path    = "C:/vault/data"
}


listener "tcp" {
  address     = "127.0.0.1:8200"
  tls_disable = 0
  tls_min_version = "tls12"
  tls_cert_file = "vault.crt"
  tls_key_file = "vault.key"
  
}

plugin_directory = "C:/vault/vault_plugins"

api_addr = "https://127.0.0.1:8200"
ui = true

log_level = "Trace"
```

**Additional context**
-

**Comment by michelvocks** [comment]
Hi @Obito93!

Thank you for submitting this bug report and sorry for the late reply.
Would you mind to provide the command used to register the plugin at the plugin-catalog?

Cheers,
Michel

**Comment by Obito93** [comment]
Hello @michelvocks,
Unfortunately my internship is finished and I am no longer part of the project that worked with the vault. I worked around the bug by setting up a linux server on which I installed vault and tested our custom plugin there, which worked fine. I know that I used the same commands on both systems.

Sorry for not being able to give you any more help!
Merry holidays and kind regards,
Obito
